Week-5/exercise_5_3/index.js

```diff
@@ -0,0 +1,4 @@
+const hasDuplicate = arr => new Set(arr).size !== arr.length;
```

Comment:
do we need to create the whole array? Cant we return early

Reply:
yes we can